src/pages/index.md

```diff
@@ -1,54 +1,70 @@
 
-### Discover
+![extensibility strategy](./_images/extensibility-strategy-overview.png)
 
-Use these sections to learn about developing Commerce apps.
+Extensibility generally has two variations, in-process extensibility and out-of-process extensibility. In the context of Adobe Commerce, in-process extensibility refers to extended functionality offerings that are located inside or alongside the Adobe Commerce monolith. Conversely, out-of-process extensibility refers to extended functionality and services that operate outside the core software.
 
-<DiscoverBlock slots="link, text"/>
+Out-of-process extensibility can reduce the total cost of ownership by simplifying upgrades and giving developers more control of the timing, coding, and implementation of their systems.
 
-[App Developer Guide](app-development/index.md)
+Other benefits of out-of-process extensibility include:
 
-Learn how to make the transition from developing Commerce PHP extensions to developing Out-of-Process apps.
+- Scalability - Extensions can be scaled separately from the core software, allowing for greater efficiency.
 
-<DiscoverBlock slots="link, text"/>
+- Isolation - An isolated environment means that developers can upgrade or modify their extensions however they want and whenever they want without relying on a core release.
 
-[Events](events/index.md)
+- Technological Independence - Developers can choose whichever tech stacks and coding languages that fit their needs.
 
-Make Commerce transactional data available to App Builder using Adobe I/O. You can define the events to transmit data each time an event triggers, or only under circumstances defined within configuration rules.
+<p></p>
 
-<DiscoverBlock slots="link, text"/>
+## In-process offerings
 
-[Webhooks](webhooks/index.md)
+Adobe Commerce and Magento Open Source provide the following in-process extensible offerings:
 
-Configure webhooks to synchronously send data to an external system when an event triggers and update Adobe Commerce in real time.
-<DiscoverBlock slots="link, text"/>
+- [Marketplace Extensions](https://developer.adobe.com/commerce/marketplace/guides/sellers/extensions/) - Create and sell PHP extensions in the [Adobe Commerce Marketplace](https://commercemarketplace.adobe.com)
 
-[Admin UI SDK](admin-ui-sdk/index.md)
+- [REST](https://developer.adobe.com/commerce/webapi/rest) - Use REST calls to interact with your Adobe Commerce store
 
-Extend the Commerce Admin to include the menus and pages necessary for merchants to configure and interact with your app.
+- [GraphQL](https://developer.adobe.com/commerce/webapi/graphql/) - Use GraphQL requests to get customizable responses from your Adobe Commerce store
 
-<DiscoverBlock slots="link, text"/>
+## Out-of-process offerings
+
+Adobe Commerce offers the following out-of-process offerings:
+
+- [Admin UI SDK](./admin-ui-sdk/index.md) - Customize and enhance your admin with new pages and features for your merchants
```

Comment:
```suggestion
- [Admin UI SDK](./admin-ui-sdk/index.md) - Customize and enhance your Admin with new pages and features for your merchants
```

Since you have full sentences in the list, I think they should all end with periods.